testsuite/integration-arquillian/tests/base/src/test/resources/log4j.properties

```diff
@@ -89,8 +89,8 @@ log4j.logger.org.keycloak.services.clientregistration.policy=debug
 # Enable logs the JDBC parameters passed to a query
 #log4j.logger.org.hibernate.type=trace
 
-# log4j.logger.org.keycloak.models.map=trace
-# log4j.logger.org.keycloak.models.map.transaction=debug
+log4j.logger.org.keycloak.models.map=trace
+log4j.logger.org.keycloak.models.map.transaction=debug
 
 # Enable to log short stack traces for log entries enabled with StackUtil.getShortStackTrace() calls
-# log4j.logger.org.keycloak.STACK_TRACE=trace
+log4j.logger.org.keycloak.STACK_TRACE=trace
```

Comment:
GHA failed on out of memory because of these lines, better keep them commented out :-)